### PR TITLE
*: pass `connection_id` and `connection_alias` to tiflash

### LINKED ISSUE #48470: Tiflash needs connection id in logs for the convenient of debugging
## Feature Request

**Is your feature request related to a problem? Please describe:**

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What problem does this PR solve?

Issue Number: close #48470

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [x] No need to test
  > - [ ] I checked and no code files have been changed.
  >   

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -313,3 +313,5 @@ replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
 	github.com/pingcap/tidb/pkg/parser => ./pkg/parser
 )
+
+replace github.com/pingcap/kvproto => github.com/xzhangxian1008/kvproto v0.0.0-20231110025204-ddcf2223e2aa
```
Comment (SeaRise) [inline go.mod:317]: Is this expected?

Comment (xzhangxian1008) [reply go.mod:317]: > Is this expected?

yes, I will change it to the latest tipb commit before pr is merged.

File: go.sum [hunk 2]
```diff
@@ -13,39 +14,812 @@ cloud.google.com/go v0.56.0/go.mod h1:jr7tqZxxKOVYizybht9+26Z/gUq7tiRzu+ACVAMbKV
 cloud.google.com/go v0.57.0/go.mod h1:oXiQ6Rzq3RAkkY7N6t3TcE6jE+CIBBbA36lwQ1JyzZs=
 cloud.google.com/go v0.62.0/go.mod h1:jmCYTdRCQuc1PHIIJ/maLInMho30T/Y0M4hTdTShOYc=
 cloud.google.com/go v0.65.0/go.mod h1:O5N8zS7uWy9vkA9vayVHs65eM1ubvY4h553ofrNHObY=
+cloud.google.com/go v0.72.0/go.mod h1:M+5Vjvlc2wnp6tjzE102Dw08nGShTscUx2nZMufOKPI=
```
Comment (windtalker) [inline go.sum:17]: Why so many changes in this file?

Comment (xzhangxian1008) [reply go.sum:17]: > Why so many changes in this file?

These changes are temporary, I will correct it before merge.

**Review by windtalker — APPROVED** [review]
LGTM

File: go.sum (hunk 2 above)
Comment (windtalker) [reply go.sum:17]: OK

**Review by SeaRise — APPROVED** [review]

**Pull request merged**